client/src/components/ObjectEditor.vue

```diff
@@ -103,13 +103,18 @@
                 this.$store.dispatch('setAllowGlobalHotkeys', true);
             },
 
-            handleEditablePropKeyDown: function(e, prop) {
+            handleEditablePropKeyDown: function(e, prop, targetObj) {
                 if (e.key === 'Enter') {
                     e.stopPropagation();
                     e.preventDefault();
                     this.animateDivWithClass(e.target, 'success');
                     e.target.blur();
-                    this.setProperty(prop.name, e.target.innerHTML);
+                    
+                    if (targetObj.objectType == "room") {
+                      this.setProperty(prop.name, e.target.innerHTML, targetObj.textCoords);
```

Comment:
Instead of `targetObj.textCoords`, you can access `objectEditorData` directly and do this with:

```
this.objectEditorData.textCoords
```